index.js

```diff
@@ -152,30 +192,15 @@ function sendNotification(endpoint, params) {
 
       if (vapid && !isGCM) {
         // VAPID isn't supported by GCM.
+        vapid.audience = urlParts.protocol + '//' + urlParts.hostname;
```

Comment:
> This does open the question of how should a developer find out what the correct audience is for a push service? Let's say a new browser Y started support push, the audience should be....?

Maybe let's wait for https://github.com/webpush-wg/webpush-vapid/issues/23 to be addressed and then we can decide how to proceed.